libraries/SubGhz/README.md

```diff
@@ -0,0 +1,134 @@
+
+```
+SubGhz.setNssActive(true);
+```
+
+#### SPI access
+Access to the dedicated SPI bus happens through the `SubGhz.SPI` object.
+This behaves just like the normal Arduino SPI object (it is in fact
+a customized subclass of the normal SPIClass, provided by the normal SPI
+library).
+
+If you use the `beginTransaction()` method, you can pass the
+`SubGhz.spi_settings` variable to get the right settings and run at the
+maximum supported speed. If not using `beginTransaction()`, the default
+settings are also fine, just a bit slower.
+
+#### "GPIO" signals
+Some of the signals which are normally (on an actual external SX126x
+radio chip) available externally and controlled by GPIO are now
+internally connected to registers in the RCC and PWR modules. These
+signals (Nss, Reset and Busy) can be controlled and/or read through this
+library.
+
+The Nss signal can be written (and read back) using
+`SubGhz.setNssActive(bool)` and `SubGhz.isNssActive()`, the Reset signal
+using `SubGhz.setResetActive(bool) and `SubGhz.isResetActive()` and the
+busy signal can only be read using `SubGhz.isBusy()`.
+
+There is no need to configure these signals before using them (i.e.
+nothing like `pinMode()` is needed).
+
+Note that there is no way to read the DIO signals directly, since they
+are not directly available to the MCU anywhere (only through the
+interrupt controller, so with some care the DIO signal status could be
+derived from the interrupt pending status if needed).
+
+#### Interrupts
+In addition, the DIO signals produced by the radio module are connected
+together in an "OR" configuration and drive the radio interrupt. This
+interrupt can be enabled and configured using various methods in this
+library.
+
+The interrupt can be attached (and enabled) using the
+`SubGhz.attachInterrupt(callback)` method. You can pass any callable here that
+`std::function` accepts (global function, method bound with `std::bind`,
+callable object or a lambda). Attaching an interrupt clears any
+previously pending interrupts and enables the interrupt.
+
+The `SubGhz.detachInterrupt()` method can be used to disable the
+interrupt and clear the handler and the `SubGhz.hasInterrupt()` method
+can be used to query if a handler was attached (regardless of whether it
+is currently enabled).
+
+The `SubGhz.disableInterrupt()` and `SubGhz.enableInterrupt()` method
+can be used to temporarily disable the interrupt. If the interrupt is
+triggered while it is disabled, it will become pending (indicated by the
+`SubGhz.isInterruptPending()` method) and the interrupt handler will run
+directly when the interrupt is enabled again (unless cleared with
+`SubGhz.clearPendingInterrupt()`). Note that there is no method to query
+whether the interrupt is currently enabled, as the interrupt controller
+hardware does allow reading back this value.
+
+Note that since these DIO signals are always connected to the interrupt
+controller, drive a single interrupt and are not available on any
```

Comment:
`interrupt and are not available` missing words?

Reply:
It's actually a list of three things: The signals are always connected, the signals drive a single interrupt (not three separate ones) and the signals are not available anywhere else, but I can see how this is confusing. I'll reword to  be more clear.